src/main/java/club/sk1er/patcher/config/PatcherConfig.java

```diff
@@ -71,6 +71,15 @@
     )
     public static boolean fixedAlexArms = true;
 
+    @Property(
+        type = PropertyType.SELECTOR, name = "Keyboard layout",
```

Comment:
missed this, name should be "Keyboard Layout"

Reply:
Done